src/stream/src/executor/sink.rs

```diff
@@ -88,6 +91,8 @@ impl<F: LogStoreFactory> SinkExecutor<F> {
         sink_param: SinkParam,
         columns: Vec<ColumnCatalog>,
         log_store_factory: F,
+        chunk_size: usize,
+        input_data_types: Vec<DataType>,
```

Comment:
Maybe we can get the `input_data_types` from `executor.schema()` instead of passing it separatedly?

Reply:
after https://github.com/risingwavelabs/risingwave/pull/15167 I think it is banned